packages/contracts/src/contracts/protocol/rsk.ts

```diff
@@ -97,4 +97,20 @@ export const rsk: Record<string, AsyncContractConfigData> = {
     address: '0x6496DF39D000478a7A7352C01E0E713835051CcD',
     getAbi: async () => (await import('../../abis/governor.json')).default,
   },
+  governorVaultOwner: {
+    address: '0x05f4f068DF59a5aA7911f57cE4f41ebFBcB8E247',
+    getAbi: async () => [],
+  },
+  governorVaultAdmin: {
+    address: '0x51C754330c6cD04B810014E769Dab0343E31409E',
+    getAbi: async () => [],
+  },
+  adoptionFund: {
+    address: '0x0f31cfd6aAb4d378668Ad74DeFa89d3f4DB26633',
+    getAbi: async () => [],
+  },
+  developmentFund: {
+    address: '0x617866cC4a089c3653ddC31a618b078291839AeB',
+    getAbi: async () => [],
+  },
```

Comment:
In my opinion these should be moved somewhere in the bitocracy config mappings, we don't need abi's of these contracts and we also need to link them to either owner or admin governor contracts.
Check with @tiltom for best place to put them, I believe he also has list of contracts in his task

Reply:
for now, left it as it is, will update once we have response from @tjcloa [here](https://sovryn.atlassian.net/browse/SOV-2925?focusedCommentId=20484)